src/main/domain/Logger.js

```diff
@@ -0,0 +1,38 @@
+export class Logger {
+  constructor({level, key}) {
+    this._key = key
+    this._level = level
+    this._console = console
+  }
+
+  debug(log) {
+    if (this._level.shouldLog('debug')) {
+      this._log({
+        level: this._level.getLabel('debug'),
+        writter: this._console.log,
+        log
+      })
+    }
+  }
+
+  error(log) {
+    if (this._level.shouldLog('error')) {
+      this._log({
+        level: this._level.getLabel('error'),
+        writter: this._console.error,
+        log
+      })
+    }
+  }
+
+  _log({level, writter, log}) {
+    try {
+      const label = `${level} | ${this._key} | `
```

Comment:
⚠️ what would happen if I just initialize a logger without a `key`?

Reply:
@alextremp Should we let this to happen? I have an option of throwing an error if Logger is initialized without a key or I have the option of setting to key a default value, as an empty string or a "nonKeyed".